generated/nidigital/nidigital/_library.py

```diff
@@ -434,13 +434,13 @@ def niDigital_GetChannelName(self, vi, index, name_buffer_size, name):  # noqa:
                 self.niDigital_GetChannelName_cfunc.restype = ViStatus  # noqa: F405
         return self.niDigital_GetChannelName_cfunc(vi, index, name_buffer_size, name)
 
-    def niDigital_GetChannelNameFromString(self, vi, index, name_buffer_size, name):  # noqa: N802
+    def niDigital_GetChannelNameFromString(self, vi, indices, name_buffer_size, names):  # noqa: N802
```

Comment:
is this `index` to `indices` to be consistent with other ADEs (like LabVIEW) and other APIs (like NI-SCOPE)?

Reply:
No, it's to be consistent with the use of "names" in method name - `get_channel_names`. I expect it to be consistent across all bindings in nimi-python but not with LabVIEW and C where `index` is used, to go with function/VI name - `GetChannelNameFromString`.

I'm ok with using `index` in Python too, for the sake of consistency.

Reply:
I'm ok either way

Reply:
I'll keep it as `indices`